doc/classes/HTTPRequest.xml

```diff
@@ -4,8 +4,8 @@
 		A Node with the ability to send HTTP requests.
 	</brief_description>
 	<description>
-		A Node with the ability to send HTTP requests. Uses a [HTTPClient] internally, supports HTTPS.
-		Can be used to make HTTP requests or download files via HTTP.
+		A Node with the ability to send HTTP requests. Uses a [HTTPClient] internally, supports HTTPS/SSL.
```

Comment:
- Don't capitalize "Node"

- "Uses a [HTTPClient] internally, supports HTTPS/SSL."
a -> an
The comma is grammatically incorrect. I'd just drop it entirely, since the SSL support is part of HTTPClient and described there.